.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+[*.{js,json,yml}]
```

Comment:
nit: I'd say nix this part since it's the same `indent_size`

Reply:
cool, since I already merged I just updated that on master